language-support/java/bindings-rxjava/BUILD.bazel

```diff
@@ -66,16 +66,28 @@ da_scala_library(
 )
 
 da_scala_test_suite(
-    name = "bindings-java-tests",
+    name = "bindings-scala-tests",
     srcs = glob([
         "src/test/**/*Spec.scala",
         "src/test/**/*Test.scala",
     ]),
     deps = [
+        ":bindings-scala-tests-lib",
         ":bindings-java-tests-lib",
     ] + testDependencies,
 )
 
+da_java_library(
```

Comment:
I don't speak Bazel, so please double check this.

Reply:
If the intent of the following code is to emulate code generation, I would say the tests would not longer ensure we're not breaking codegen, so I would suggest to not use this approach. More on this in a later comment.